Panel.c

```diff
@@ -386,7 +386,11 @@ bool Panel_onKey(Panel* this, int key) {
       case KEY_LEFT:
       case KEY_CTRL('B'):
          if (this->scrollH > 0) {
-            this->scrollH = MAXIMUM(this->scrollH - CRT_scrollHAmount, 0);
+            if (this->scrollH > CRT_scrollHAmount) {
+               this->scrollH = this->scrollH - CRT_scrollHAmount;
+            } else {
+               this->scrollH = 0;
+            }
```

Comment:
This may be changed to a `saturatingSub` call if there is no [missed optimization](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=118679).